tests/gis_tests/rasterapp/test_rasterfield.py

```diff
@@ -1,20 +1,47 @@
 import json
 
+from django.contrib.gis.db.models.lookups import (
+    DistanceLookupBase, gis_lookups,
+)
+from django.contrib.gis.geos import GEOSGeometry
+from django.contrib.gis.measure import D
 from django.contrib.gis.shortcuts import numpy
 from django.core.exceptions import ImproperlyConfigured
+from django.db.models import Q
 from django.test import (
     TestCase, TransactionTestCase, mock, skipUnlessDBFeature,
 )
 
 from ..data.rasters.textrasters import JSON_RASTER
 from ..models import models
-from .models import RasterModel
+from .models import RasterModel, RasterRelatedModel
 
 
 @skipUnlessDBFeature('supports_raster')
 class RasterFieldTest(TransactionTestCase):
     available_apps = ['gis_tests.rasterapp']
 
+    def setUp(self):
+        from django.contrib.gis.gdal import GDALRaster
```

Comment:
How about this at the top of the file:
```python
try:
    from django.contrib.gis.gdal import GDALRaster
except ImportError: # or whatever exception it is
    GDALRaster = None
```
to avoid the duplicated inner imports?